apps/lndhub/docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+version: "3.7"
+
+services:
+        redis:
+              container_name: redis
+              image: redis:6.2-buster
+              command: redis-server --requirepass moneyprintergobrrr
+              restart: on-failure
+              stop_grace_period: 5m
+              stop_signal: SIGINT
+              volumes:
+                  - ${APP_DATA_DIR}/redis:/data
+              networks:
+                  default:
+                      ipv4_address: $APP_LNDHUB_REDIS_IP
+        web:
+                container_name: web
+                image: bluewalletorganization/lndhub:v1.3.0
```

Comment:
This image is arm64 only: https://hub.docker.com/layers/bluewalletorganization/lndhub/v1.3.0/images/sha256-2e0785e93ff328f1b33521573e5725606c10d8654c9c6dd0a557469890a1c398?context=explore

Umbrel needs multi-arch builds supporting at least arm64 and amd64.

Reply:
Looks like both archs are being built but they're being pushed over the same tag: https://github.com/BlueWallet/LndHub/blob/f64a8c8102b19e87ea3f9f877e7ba27976dcc304/.github/workflows/tag.yml

@AaronDewes you need to do a single buildx that builds both archs instead of doing two buildx runs of separate archs with the same tag.

Reply:
Sorry about that, I thought multi-arch separate builds were faster.